share/tasks.py

```diff
@@ -208,7 +208,7 @@ def do_run(self, *args, **kwargs):
                 cg = ChangeGraph(self.normalized.data['@graph'], namespace=self.normalized.source.username)
                 cg.process()
                 cs = ChangeSet.objects.from_graph(cg, self.normalized.id)
-                if cs and (self.source.is_robot or self.source.is_trusted):
+                if cs and (self.source.is_robot or self.source.is_trusted or self.source.username == settings.APPLICATION_USERNAME):
```

Comment:
That's what I did at first, but `system` isn't a superuser. Should the `merge` command submit its changes as `admin`?